orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/GetCommitsTask.groovy

```diff
@@ -150,7 +150,8 @@ class GetCommitsTask implements DiffTask {
         clusterPair?.canary?.availabilityZones?.findResult { key, value -> key == region }
       }?.canary?.amiName
     } else if (context.deploymentDetails) { // deploy asg stage
-      return context.deploymentDetails.find { it.region == region }?.amiId
+      def details = context.deploymentDetails.find { it.region == region }
+      return details?.amiId ?: details?.ami
```

Comment:
@tomaslin Not that I recall, other than confusion around what `ami` and `amiId` are meant to represent. 

For the purpose of a standard deploy, I believe clouddriver is smart enough to use either an ami name or id ... which might explain why this has worked all this time (or it's using `imageId` instead).

I have no problem with `AmazonImageDetails` becoming:

```
put("ami", imageId)
```